### ISSUE TITLE
Handle some GDrive errors directly from UI

### ISSUE THREAD

**Issue by dzek69** [hwittenborn/celeste#69]
This is probably blocked by #59 - it requires Celeste to have any rclone/servers settings in the UI first, but I report it while I remember

I have a file that Google considers "abusive", so syncing ends with the error:
![image](https://user-images.githubusercontent.com/4936805/221409974-f025c560-1249-4d98-8750-7a41250dae11.png)

Error message suggests to add a flag to rclone to confirm I want abusive files to be downloaded on my computer. While this is currently impossible from UI - after it will become possible this exact error on click could ask user if they want this config setting to be automatically changed for them.

**Comment by hwittenborn** [comment]
Do you think it'd make sense to always just leave that flag enabled? Celeste usually just syncs personal files, so users would probably always consider those kind of files safe. What do you think?

**Comment by dzek69** [comment]
I don't think it should be enabled by default, because if I had some shared folder (I shared some folder with write access or just pinned somebody's folder into my drive) other people could push a virus to it (even by accident, because their PC got infected) that I may not want on my PC.

That's a risk that should be prevented by default and only clear statement that I know the risks should enable that.

Ideally it would ask me once for each new file with the issue, but I guess this is something that only rclone can implement? I don't really know what exactly Celeste is responsible for and what is a rclone responsibility, sorry.

**Comment by hwittenborn** [comment]
> I don't think it should be enabled by default, because if I had some shared folder (I shared some folder with write access or just pinned somebody's folder into my drive) other people could push a virus to it (even by accident, because their PC got infected) that I may not want on my PC.

I didn't think about that, that's fair enough logic for me.

> Ideally it would ask me once for each new file with the issue, but I guess this is something that only rclone can implement?

Celeste should be able to implement it, Celeste has a database file and I can just store what files have accepted the warning in there. It would probably be good to add options for one-time accepts and accepts that work forever, right?

**Comment by dzek69** [comment]
Yeah, sounds great!
